lib/google/gapic/generator/template_provider.rb

```diff
@@ -0,0 +1,40 @@
+# Copyright 2018 Google LLC
```

Comment:
We don't need this. (Ruby tends not to use abstract base classes or interfaces, since there are no static types.)

Reply:
Removed the abstract base class.